examples/apps/data-object-grid/tsconfig.json

```diff
@@ -1,10 +1,9 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": "../../../common/build/build-common/tsconfig.base.json",
 	"compilerOptions": {
-		"module": "esnext",
-		"target": "es6",
-		"outDir": "lib",
-		"jsx": "react",
+		"module": "ESNext",
+		"moduleResolution": "Bundler",
+		"outDir": "./lib",
```

Comment:
My first attempt to was to re-export the `<Collapsible>` react control from a *.cts file.  TypeScript liked that, but webpack wouldn't resolve the *.cjs file.

I then switched to ESNext/Bundler, which both WebPack and TypeScript were happy with.

Reply:
Hmm. I am suspicious. Webpack should be happy with .cjs and .mjs. Those are common enough files now. I suspect that our configuration is incorrect.
https://webpack.js.org/configuration/resolve/
shows and example where .mjs is handled completely apart from .js. I suspect .cjs should be the same.

Reply:
webpack will succeed with this configuration:
```json
				extensionAlias: {
					".cjs": [".cts", ".cjs"],
					".js": [".ts", ".tsx", ".js"],
					".mjs": [".mts", ".mjs"],
				},
```

Reply:
I am confused why we would extend from build-common/tsconfig.node16.json (which is all about Node16) when we override the two settings it provides. Let's avoid that.
cc @tylerbutler

Reply:
Used the .cts workaround so this could build with Node16 and updated webpack in #20062